contracts/src/v0.8/ccip/test/pools/GHO/GhoTokenPoolEthereum.t.sol

```diff
@@ -0,0 +1,996 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import {TransparentUpgradeableProxy} from "solidity-utils/contracts/transparent-proxy/TransparentUpgradeableProxy.sol";
+import {stdError} from "forge-std/Test.sol";
+import {IERC165} from "../../../../vendor/openzeppelin-solidity/v4.8.3/contracts/utils/introspection/IERC165.sol";
+import {ILiquidityContainer} from "../../../../liquiditymanager/interfaces/ILiquidityContainer.sol";
+import {IPoolV1} from "../../../interfaces/IPool.sol";
+import {Pool} from "../../../libraries/Pool.sol";
+import {LockReleaseTokenPool} from "../../../pools/LockReleaseTokenPool.sol";
+import {UpgradeableLockReleaseTokenPool} from "../../../pools/GHO/UpgradeableLockReleaseTokenPool.sol";
+import {UpgradeableTokenPool} from "../../../pools/GHO/UpgradeableTokenPool.sol";
+import {EVM2EVMOffRamp} from "../../../offRamp/EVM2EVMOffRamp.sol";
+import {RateLimiter} from "../../../libraries/RateLimiter.sol";
+import {MockUpgradeable} from "../../mocks/MockUpgradeable.sol";
+import {GhoTokenPoolEthereumSetup} from "./GhoTokenPoolEthereumSetup.t.sol";
+
+contract GhoTokenPoolEthereum_setRebalancer is GhoTokenPoolEthereumSetup {
+  error OnlyCallableByOwner();
```

Comment:
is it worth to import this error from the corresponding contract?  same with `SenderNotAllowed`

Reply:
wdyt?

Reply:
I like it! thanks